homeassistant/components/netgear_lte/__init__.py

```diff
@@ -23,10 +26,25 @@
 DOMAIN = 'netgear_lte'
 DATA_KEY = 'netgear_lte'
 
+
+NOTIFY_SCHEMA = vol.Schema({
+    vol.Optional(CONF_NAME, default=DOMAIN): cv.string,
+    vol.Optional(CONF_RECIPIENT, default=[]):
+        vol.All(cv.ensure_list, [cv.string]),
+})
+
+SENSOR_SCHEMA = vol.Schema({
+    vol.Required(CONF_MONITORED_CONDITIONS):
+        vol.All(cv.ensure_list, [cv.string]),
+})
+
 CONFIG_SCHEMA = vol.Schema({
     DOMAIN: vol.All(cv.ensure_list, [vol.Schema({
         vol.Required(CONF_HOST): cv.string,
         vol.Required(CONF_PASSWORD): cv.string,
+        vol.Optional(NOTIFY_DOMAIN, default=NOTIFY_SCHEMA({})):
```

Comment:
Default values will be passed through the corresponding schema on eval of the main schema. So we only need to put an empty dict as default value here.